lib/events/firestoreevents/README.md

```diff
@@ -74,14 +74,6 @@ the `purgeExpiredDocuments` property, which defaults to true. Purging is done ba
 configurable `eventRetentionPeriod` property, which defaults to a year. Add this property to
 the URI to change the retention period.
 
-Two composite indexes are required for this implementation:
-
-1. `EventNamespace` ascending, then on `CreatedAt` ascending
-2. `SessionID` ascending, then on `EventIndex` ascending
-
-Composite indexes should be limited to the specific collection set in the
-configuration (in the aforementioned example is `events`).
```

Comment:
Can you help me understand why this section is no longer needed?

Reply:
It had already been out of date prior to my changes. Instead of updating to describe all the current indexes, I decided to remove this section entirely as it's pretty probable for it to become incorrect again with the upcoming refactoring changes. It's much more reliable, and not that more difficult, to glean this information directly from the code (the function `ensureIndexes` in `firestoreevents.go`) or by running a test Teleport auth server configured with Firestore and then looking at the created indexes in the GCP console.